DOCSP-43061 Stable API (#56)
mongodb/docs-c: source/connect/stable-api.txt

.. literalinclude:: /includes/connect/stable-api.c
   :language: c
   :start-after: // start-stable-api
   :end-before: // end-stable-api

Once you create a client instance with
a specified API version, all commands you run with the client use the specified
version. If you need to run commands using more than one version of the 
{+stable-api+}, create a new client.

.. _stable-api-options:

Configure the {+stable-api+}
------------------------

The following table describes the parameters of the ``ServerApi`` class. You can use these
parameters to customize the behavior of the {+stable-api+}.

.. list-table::
   :header-rows: 1
   :stub-columns: 1
   :widths: 25,75

   * - Option Name
     - Description

   * - strict
     - | **Optional**. When ``true``, if you call a command that isn't part of 
         the declared API version, the driver raises an exception.
       |
       | Default: **false**

   * -  deprecation_errors
     - | **Optional**. When ``true``, if you call a command that is deprecated in the 
         declared API version, the driver raises an exception.
       |
       | Default: **false**

> The following code example shows how you can use these parameters when constructing a 
> ``ServerApi`` object:

Reviewer: ```suggestion
The following code example shows how you can set both options to ``true`` when constructing a 
``ServerApi`` object:
```
